proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -120,6 +139,8 @@ message CuratedPlugin {
   RegistryConfig registry_config = 12;
   // The revision for this plugin version.
   uint32 revision = 13;
+  // The languages supported by the plugin.
+  repeated PluginLanguage languages = 14;
```

Comment:
how do you think if we explicitly call this `output_languages`?

Reply:
Sure.